tests/bson/bson-toPHP_error-004.phpt

```diff
@@ -35,13 +35,13 @@ foreach ($tests as $bson) {
 <?php exit(0); ?>
 --EXPECTF--
 OK: Got MongoDB\Driver\Exception\UnexpectedValueException
-Could not convert BSON document to a PHP variable
+Detected corrupt BSON data.
```

Comment:
I believe you forgot to remove the trailing period in these expected exception messages. Fixed in https://github.com/mongodb/mongo-php-driver/commit/e6deaeb59ae24120edcf15f9dbd6ad7357c164fc.